Hi3Helper.Core/Classes/DiscordPresence/DiscordPresenceManager.cs

```diff
@@ -162,11 +196,11 @@ public async void SetActivity(ActivityType activity, int delay = 500)
             }
         }
 
-        private void BuildActivityGameStatus(string activityName)
+        private void BuildActivityGameStatus(string activityName, bool isGameStatusEnabled)
         {
             _activity = new Activity
             {
-                Details = $"{activityName} {ConfigV2Store.CurrentConfigV2GameCategory}",
+                Details = $"{activityName} {(!isGameStatusEnabled ? ConfigV2Store.CurrentConfigV2GameCategory : "")}",
```

Comment:
When the toggle for using Game Name as Activity Name is on, I think instead of making this string empty we can use it to advertise our Launcher with a localizable string.
Something like "Using Collapse Launcher" or something